test/testModel.cc

```diff
@@ -1099,8 +1099,8 @@ SUITE(Integrate)
       model->addItem(a);
       model->addWire(new Wire(intop->ports[0],a->ports[1]));
       intop->description("a");
-      // should cowardly refuse, and give a different name
-      CHECK_EQUAL("a1",intop->description());
+      // should cowardly refuse, and give a different name. Wynand: I don't really understand this. why can't the description be the same as the intop name?
```

Comment:
Because there is already a flow variable called "a". If it didn't "cowardly refuse", then the model ends up in an inconsistent state of a being both a flow variable and and integral variable.